src/main/resources/application.properties

```diff
@@ -47,3 +48,12 @@ app.cleanup.scheduling=604800000
 # every 6 hours = 1000*60*60*6
 # note: dictates both fixed rate and initial delay
 app.cache-evict.scheduling=21600000
+
+# Repository cloning Parameters
+# app.crawl.cloning.enabled: whether the CodeAnalysis job is enabled
+app.crawl.cloning.enabled=true
+app.crawl.cloning.folderprefix=ghs-cloned-
+# app.crawl.cloning.scheduling: in milliSecond # every 6 hours
+app.crawl.cloning.scheduling=21600000
+# app.crawl.cloning.maxpoolthreads: the maximum amount of worker threads dedicated to cloning and computing code metrics
+app.crawl.cloning.maxpoolthreads=8
```

Comment:
Maybe instead of using `app.crawl.cloning`, rename this property prefix to `app.analysis`